engine/src/em-event.js

```diff
@@ -27,57 +27,76 @@ mergeInto(LibraryManager.library, {
 		// contains mouse event data
 		_mouseEvent: null,
 
+		// text element used to capture input events
+		_inputelement: null,
+
 		// This function is used to call a function for each event
 		// type to handler mapping defined for LiveCode on Emscripten.
 		//
-		// The <func> must take two arguments: the event type, and the
-		// handler function used to handle that event type.
+		// The <func> must take three arguments: the event type, the
+		// handler function used to handle that event type, and a
+		// boolean indicating whether the event handlershould be
```

Comment:
`the event handler should` - missing space

Reply:
Typo fixed :)